EnergyCorrelatorOptions.h

```diff
@@ -60,11 +60,15 @@ namespace EnergyCorrelatorOptions {
 
   // Smearing options
   const bool modCsts = true;
-  const float theta = 1;
-  const float pTSmear = 0;
+  const float theta = 0;
+  const float pTSmear = 0.02;
   const float effVal = 1;
 
-
+  //Manual calculation option
+  const bool doManualCalc = true;
+  const int mom_option = 0;
+  const int norm_option = 0;
```

Comment:
I'd suggest using the enum itself here!

```suggestion
  const int mom_option = SEnergyCorrelator::Norm::E;
  const int norm_option = SEnergyCorrelator::Norm::E;
```